deploy/nginx.conf

```diff
@@ -75,4 +75,8 @@ server {
     location / {
       proxy_pass http://127.0.0.1:8081;
     }
+
+    location /output {
+      alias /var/www/trypurescript/staging/.psci_modules/node_modules;
```

Comment:
Is it a concern that this also potentially serves very large externs? Is it possible to glob only js files?

Reply:
That's probably not a bad idea. I think it should be possible, I'll play around with it a bit.

Reply:
Yep, this is pretty straightforward. There's even an example in the docs which covers pretty much this exact use case: http://nginx.org/en/docs/http/ngx_http_core_module.html#location